2022/typescript/day05.ts

```
@@ -1,41 +1,41 @@
 
 const getInput = (): { stacks: { [key: number]: string[] }; moves: any[] } => {
-  const [current, instructions] = Library.getInput("day05").split(/\n\s1.*\n\n/);
+  const [current, instructions] = Library.getInput('day05').split(/\n\s1.*\n\n/)
 
-  const stacks: any = {};
-  current.split("\n").forEach((line) => {
+  const stacks: any = {}
+  current.split('\n').forEach((line) => {
     for (let i = 1, j = 1; i < line.length; i += 4, j++) {
-      if (line[i].trim()) stacks[j] ? stacks[j].push(line[i]) : (stacks[j] = [line[i]]);
+      if (line[i].trim()) stacks[j] ? stacks[j].push(line[i]) : (stacks[j] = [line[i]])
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)